pythonfmu/pythonfmu-export/cpp/PyObjectWrapper.cpp

```diff
@@ -62,9 +62,12 @@ PyObjectWrapper::PyObjectWrapper(const std::string& instanceName, const std::str
     if (pClass == nullptr) {
         handle_py_exception("[ctor] PyObject_GetAttr");
     }
-    PyObject *argList = Py_BuildValue("s", instanceName.c_str());
-    pInstance_ = PyObject_CallFunctionObjArgs(pClass, argList, nullptr);
-    Py_DECREF(argList);
+
+    PyObject* args = Py_BuildValue("s", instanceName.c_str());
+    PyObject* kwargs = Py_BuildValue("{s:i}", "visible", visible);
```

Comment:
Nice to know that a dictionary does it for kwargs.

```suggestion
PyObject* kwargs = Py_BuildValue("{sssi}", "instance_name", instanceName.c_str(), "visible", visible);
```

`:` should not be present